lib/go/model_operator_stats.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * REST API
+ *
+ * Rockset's REST API allows for creating and managing all resources in Rockset. Each supported endpoint is documented below.  All requests must be authorized with a Rockset API key, which can be created in the [Rockset console](https://console.rockset.com). The API key must be provided as `ApiKey <api_key>` in the `Authorization` request header. For example: ``` Authorization: ApiKey aB35kDjg93J5nsf4GjwMeErAVd832F7ad4vhsW1S02kfZiab42sTsfW5Sxt25asT ```  All endpoints are only accessible via https.  Build something awesome!
+ *
+ * API version: v1
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package rockset
+import (
+    "bytes"
+    "encoding/json"
+    "fmt"
+    
+)
+
+type OperatorStats struct {
```

Comment:
I will hold on for now as we might not want to expose `OperatorStats` to user. I will get back to you on this.

Reply:
just discussed internally, should be fine to expose this.